miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -323,13 +342,16 @@ export.validate_id
     push.MIN_ACCOUNT_ONES u32gte assert.err=ERR_ACCOUNT_INSUFFICIENT_NUMBER_OF_ONES
 end
 
-#! Sets the code of the account the transaction is being executed against. This procedure can only be
-#! executed on regular accounts with updatable code. Otherwise, this procedure fails.
+#! Sets the code of the account the transaction is being executed against.
 #!
-#! Stack: [CODE_COMMITMENT]
-#! Output: []
+#! Inputs:  [CODE_COMMITMENT]
+#! Outputs: []
 #!
+#! Where:
 #! - CODE_COMMITMENT is the hash of the code to set.
+#!
+#! Panics if:
+#! - this procedure is executed on the account which type differs from the `basic mutable`.
```

Comment:
I used the notation we're using in the docs [here](https://docs.polygon.technology/miden/miden-base/architecture/accounts/#account-types), but I'm fine with other options, I think it is more important to make types consistent rather than use some specific term.